nemo/collections/asr/parts/utils/transcribe_utils.py

```diff
@@ -11,15 +11,295 @@
 from nemo.collections.asr.models.ctc_models import EncDecCTCModel
+from nemo.collections.asr.parts.utils import rnnt_utils
+from nemo.collections.asr.parts.utils.streaming_utils import FrameBatchASR
 from nemo.utils import logging
 
 
+def get_buffered_pred_feat_rnnt(
+    asr: FrameBatchASR,
+    tokens_per_chunk: int,
+    delay: int,
+    model_stride_in_secs: int,
+    batch_size: int,
+    manifest: str = None,
+    filepaths: List[list] = None,
+) -> List[rnnt_utils.Hypothesis]:
+    """
+    Moved from examples/asr/asr_chunked_inference/rnnt/speech_to_text_buffered_infer_rnnt.py
+    Write all information presented in input manifest to output manifest and removed WER calculation.
+    """
+    hyps = []
+    refs = []
+
+    if filepaths and manifest:
+        raise ValueError("Please select either filepaths or manifest")
+    if filepaths is None and manifest is None:
+        raise ValueError("Either filepaths or manifest shoud not be None")
+
+    if manifest:
+        filepaths = []
+        with open(manifest, "r") as mfst_f:
+            print("Parsing manifest files...")
+            for l in mfst_f:
+                row = json.loads(l.strip())
+                filepaths.append(row['audio_filepath'])
+                if 'text' in row:
+                    refs.append(row['text'])
+
+    with torch.inference_mode():
+        with torch.cuda.amp.autocast():
+            batch = []
+            asr.sample_offset = 0
+            for idx in tqdm(range(len(filepaths)), desc='Sample:', total=len(filepaths)):
+                batch.append((filepaths[idx]))
+
+                if len(batch) == batch_size:
+                    audio_files = [sample for sample in batch]
+
+                    asr.reset()
+                    asr.read_audio_file(audio_files, delay, model_stride_in_secs)
+                    hyp_list = asr.transcribe(tokens_per_chunk, delay)
+                    hyps.extend(hyp_list)
+
+                    batch.clear()
+                    asr.sample_offset += batch_size
+
+            if len(batch) > 0:
+                asr.batch_size = len(batch)
+                asr.frame_bufferer.batch_size = len(batch)
+                asr.reset()
+
+                audio_files = [sample for sample in batch]
+                asr.read_audio_file(audio_files, delay, model_stride_in_secs)
+                hyp_list = asr.transcribe(tokens_per_chunk, delay)
+                hyps.extend(hyp_list)
+
+                batch.clear()
+                asr.sample_offset += len(batch)
+
+    if os.environ.get('DEBUG', '0') in ('1', 'y', 't'):
+        if len(ref) == 0:
+            print("ground-truth text does not present!")
+            for hyp in hyps:
+                print("hyp:", hyp)
+        else:
+            for hyp, ref in zip(hyps, refs):
+                print("hyp:", hyp)
+                print("ref:", ref)
+
+    wrapped_hyps = wrap_transcription(hyps)
+    return wrapped_hyps
+
+
+def get_buffered_pred_feat(
```

Comment:
have bpe in function name for both, we don't support char models mostly

Reply:
I would prefer keep the name for now since we still support some char models. Could update later if we decide to deprecate the models

Reply:
Ok sounds good.